requirements/edx/base.txt

```diff
@@ -20,8 +20,7 @@
 -e openedx/core/lib/xblock_builtin/xblock_discussion  # via -r requirements/edx/local.in
 -e git+https://github.com/edx-solutions/xblock-google-drive.git@2d176468e33c0713c911b563f8f65f7cf232f5b6#egg=xblock-google-drive  # via -r requirements/edx/github.in
 -e common/lib/xmodule  # via -r requirements/edx/local.in
--e git+https://github.com/open-craft/xblock-lti-consumer.git@2.10.1-opencraft#egg=lti-consumer #via -r requirements/edx/base.in
```

Comment:
Is this change intentional?

Reply:
As I understand this is a transitive dependency pulled by a package which is now pulling lti-consumer-xblock==3.4.4 [ref](https://github.com/open-craft/edx-platform/pull/464/files#diff-160eb30f60d519b8fe8a919cb2646a7ebec7652952129cfdd803776e70be6300R154)

Reply:
@gabor-boros, yeah, I know it's replaced by that entry. I don't know why it's not pinned in `base.in`, but I also don't know if it's fully compatible with Lilac, as this is a jump from the custom`v2.10.1` to `v3.4.4`. `open-release/lilac.2` [is using](https://github.com/openedx/edx-platform/blob/open-release/lilac.master/requirements/edx/base.txt#L150) `v2.9.1`.

If we really want to bump this dependency, then we should make sure that all changes between these versions are compatible. If I remember correctly, [v3.0.0](https://github.com/openedx/xblock-lti-consumer/pull/172) needs [this edx-platform change](https://github.com/openedx/edx-platform/pull/26982), which wasn't included before Maple. We were trying to maintain backward compatibility [before](https://github.com/openedx/xblock-lti-consumer/pull/159#discussion_r633578622), but this seems to be a breaking change.

Reply:
@Agrendalath I had to upgrade pip-tools locally which resulted in a slightly different formatting of comments in the requirements files. The lti-consumer-xblock is not set back to our own version.